src/main/webapp/app/atlas/manage/taxonomy-select/taxonomy-select.component.html

```
@@ -1,4 +1,4 @@
-<select [formControl]="form" class="form-select" id="{{ selectId }}">
+<select [formControl]="form" class="form-select" id="{{ selectId() }}">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)